R/zchunk_LB123.Electricity.R

```diff
@@ -1,20 +1,19 @@
 #' module_gcam.usa_LB123.Electricity
 #'
-#' Briefly describe what this chunk does.
+#' By state, calculates electricity fuel consumption, electricity generation, and inputs and outputs of net ownuse.
```

Comment:
"ownuse"?

Reply:
ownuse is the electricity consumed by the power plants or transformation process.